azure-toolkit-libs/azure-toolkit-auth-lib/src/main/java/com/microsoft/azure/toolkit/lib/auth/Account.java

```diff
@@ -68,16 +69,19 @@ public TokenCredential getTokenCredential(String tenantId) {
     }
 
     public AzureTokenCredentials getTokenCredentialV1(String tenantId) {
+        requireAuthenticated();
         return AzureTokenCredentialsAdapter.from(getEnvironment(), tenantId, getTokenCredential(tenantId));
     }
 
     public TokenCredential getTokenCredentialForSubscription(String subscriptionId) {
-        Subscription subscription = getSubscriptionById(subscriptionId);
+        requireAuthenticated();
```

Comment:
I would suggest to rename 
`getTokenCredentialForSubscription` -> `getTokenCredential`
`getTokenCredential` -> `getTokenCredentialForTenant`
since the user would use `getTokenCredential` of subscription more frequently.

Reply:
Will be rename later